collector/indices.go

```diff
@@ -423,6 +472,18 @@ func (i *Indices) Collect(ch chan<- prometheus.Metric) {
 				metric.Value(indexStats),
 				metric.Labels(indexName)...,
 			)
+
+		}
+		if i.shards {
```

Comment:
I'd write this as `if !i.shards { continue }` so you could outdent the following loop, but it's OK to leave it as is.